cloud/cloud_manifest.cc

```diff
@@ -108,9 +102,25 @@ Status CloudManifest::LoadFromLog(std::unique_ptr<SequentialFileReader> log,
   if (!std::is_sorted(pastEpochs.begin(), pastEpochs.end())) {
     return Status::Corruption("Cloud manifest records not sorted");
   }
+  return status;
+}
+
+}  // namespace
+
+// Format:
+// header: format_version (varint) number of records (varint)
+// record: tag (varint, 1 or 2)
+// record 1: epoch (slice), file number
+// record 2: current epoch
+Status CloudManifest::LoadFromLog(std::unique_ptr<SequentialFileReader> log,
+                                  std::unique_ptr<CloudManifest>* manifest) {
+  std::string currentEpoch;                                
+  std::vector<std::pair<uint64_t, std::string>> pastEpochs;
+  auto st = LoadEpochsFromLog(std::move(log), currentEpoch, pastEpochs);
```

Comment:
Why extract this in the separate function? As far as I can tell the behavior is exactly the same.

Reply:
oops. It's introduced in previous commit. I should remove it